modules/custom/az_core/css/az-core-chosen.css

```
@@ -0,0 +1,5 @@
+.chosen-container,
```

This is one possible fix for the issue that @trackleft identified:

```suggestion
.chosen-container-single,
```

Then we could also add a fixed minimum width for multiple-select containers, something like:
```css
.chosen-container-multi {
    min-width: 250px;
}
```

See the News Tags field on AZ News nodes for an example.